allink_core/core_apps/allink_list_multicol/models.py

```diff
@@ -0,0 +1,23 @@
+from cms.models import CMSPlugin
+from django.db import models
+from djangocms_text_ckeditor.fields import HTMLField
+
+
+class AllinkListMulticolPlugin(CMSPlugin):
+    COLUMN_CHOICES = [
```

Comment:
i think it would make sense place this in a settings variable. so we could extend that n a per project level. What do you think?

Reply:
@tuerlefl i thought about that too but more than three columns are unrealistic according to Nicolai.

Reply:
even better. let's keep it simple. I will show him this comment, if I ever see 4 columns :)